jdaviz/configs/imviz/plugins/tools.py

```diff
@@ -6,49 +6,16 @@
 from glue.config import viewer_tool
 from glue_jupyter.bqplot.common.tools import Tool
 from glue.viewers.common.tool import CheckableTool
-from glue_jupyter.bqplot.common.tools import (HomeTool, BqplotPanZoomMode,
-                                              BqplotPanZoomXMode, BqplotPanZoomYMode,
-                                              BqplotRectangleMode, BqplotCircleMode,
-                                              BqplotEllipseMode, BqplotXRangeMode,
-                                              BqplotYRangeMode)
+from glue_jupyter.bqplot.common.tools import BqplotPanZoomMode
+
+from jdaviz.core.tools import BoxZoom
 
 __all__ = []
 
 ICON_DIR = os.path.join(os.path.dirname(__file__), '..', '..', '..', 'data', 'icons')
 
 
-# Override icons for built-in tools from glue-jupyter
-HomeTool.icon = os.path.join(ICON_DIR, 'home.svg')
-BqplotPanZoomMode.icon = os.path.join(ICON_DIR, 'pan.svg')
```

Comment:
Is it hard for me to see from diff but you might want to make sure that we delete icons that are no longer necessary.

Reply:
these icons are still used, the code to modify them was just moved from `imviz.plugin.tools` to `jdaviz.core.tools`.  The only unused icons currently in the repo are a yrange zoom (currently unused but I requested the icon since we may need it in the future) and an additional pan icon.